dronekit/__init__.py

```diff
@@ -184,6 +184,13 @@ def __init__(self, north, east, down):
     def __str__(self):
         return "LocationLocal:north=%s,east=%s,down=%s" % (self.north, self.east, self.down)
 
+    def distance(self):
+        """
+        Distance away from home, in meters.
+        """
+        if self.north is not None and self.east is not None and self.down is not None:
```

Comment:
Could we reformulate this so that if you don't have a component it isn't added instead?  That would allow someone who only has a 2D position to still work out where they are.

Kinda
```
sum = 0
if self.north is not None:
  sum += self.north**2
...
return math.sqrt(sum)
```